packages/angular_devkit/build_angular/src/tools/babel/plugins/adjust-typescript-enums_spec.ts

```diff
@@ -69,12 +68,45 @@ describe('adjust-typescript-enums Babel plugin', () => {
         })(ChangeDetectionStrategy || (ChangeDetectionStrategy = {}));
       `,
       expected: `
-        export var ChangeDetectionStrategy = /*#__PURE__*/ (() => {
-          ChangeDetectionStrategy = ChangeDetectionStrategy || {};
+        export var ChangeDetectionStrategy = /*#__PURE__*/ (function (ChangeDetectionStrategy) {
           ChangeDetectionStrategy[(ChangeDetectionStrategy["OnPush"] = 0)] = "OnPush";
           ChangeDetectionStrategy[(ChangeDetectionStrategy["Default"] = 1)] = "Default";
           return ChangeDetectionStrategy;
-        })();
+        })(ChangeDetectionStrategy || {});
+      `,
+    }),
+  );
+
+  it(
+    'does not wrap exported TypeScript enums from CommonJS (<5.1)',
```

Comment:
Question: Isn't this a regression? Doesn't it result in enums being less tree shakable than today?

Reply:
This case was never actually wrapped.  But it was also not broken by the transformation unlike the TS 5.1+ case.
So no regression but still suboptimal behavior for this one particular case.  However, this older TS form will become increasingly less common over time.

Reply:
Ok, that sounds reasonable. Agree we don't need to overoptimize for CommonJS.